External/Plugins/ProjectManager/Controls/Icons.cs

```diff
@@ -275,6 +275,77 @@ public static Image Overlay(Image image, Image overlay, int x, int y)
+                Images[ProjectClasspathError.Index] = ProjectClasspathError.Img;
+                Images[ClasspathError.Index] = ClasspathError.Img;
+                Images[Font.Index] = Font.Img;
+                Images[ImageResource.Index] = ImageResource.Img;
+                Images[ActionScript.Index] = ActionScript.Img;
+                Images[FlashCS3.Index] = FlashCS3.Img;
+                Images[HaxeFile.Index] = HaxeFile.Img;
+                Images[SwfFile.Index] = SwfFile.Img;
+                Images[SwfFileHidden.Index] = SwfFileHidden.Img;
+                Images[SwcFile.Index] = SwcFile.Img;
+                Images[Folder.Index] = Folder.Img;
+                Images[FolderCompile.Index] = FolderCompile.Img;
+                Images[TextFile.Index] = TextFile.Img;
+                Images[ActionScriptCompile.Index] = ActionScriptCompile.Img;
+                Images[HtmlFile.Index] = HtmlFile.Img;
+                Images[AddFile.Index] = AddFile.Img;
+                Images[OpenFile.Index] = OpenFile.Img;
+                Images[EditFile.Index] = EditFile.Img;
+                Images[Browse.Index] = Browse.Img;
+                Images[FindAndReplace.Index] = FindAndReplace.Img;
+                Images[FindInFiles.Index] = FindInFiles.Img;
+                Images[Cut.Index] = Cut.Img;
+                Images[Copy.Index] = Copy.Img;
+                Images[Paste.Index] = Paste.Img;
+                Images[Delete.Index] = Delete.Img;
+                Images[Rename.Index] = Rename.Img;
+                Images[Options.Index] = Options.Img;
+                Images[OptionsWithIssues.Index] = OptionsWithIssues.Img;
+                Images[NewProject.Index] = NewProject.Img;
+                Images[GreenCheck.Index] = GreenCheck.Img;
+                Images[Gear.Index] = Gear.Img;
+                Images[X.Index] = X.Img;
+                Images[Info.Index] = Info.Img;
+                Images[Class.Index] = Class.Img;
+                Images[Method.Index] = Method.Img;
+                Images[Variable.Index] = Variable.Img;
+                Images[Const.Index] = Const.Img;
+                Images[Icons.Refresh.Index] = Icons.Refresh.Img;
+                Images[Debug.Index] = Debug.Img;
+                Images[UpArrow.Index] = UpArrow.Img;
+                Images[DownArrow.Index] = DownArrow.Img;
+                Images[AllClasses.Index] = AllClasses.Img;
+                Images[SyncToFile.Index] = SyncToFile.Img;
+                Images[ClasspathFolder.Index] = ClasspathFolder.Img;
+                Images[LibrarypathFolder.Index] = LibrarypathFolder.Img;
+                Images[DocumentClass.Index] = DocumentClass.Img;
+                Images[CommandPrompt.Index] = CommandPrompt.Img;
+                Images[CollapseAll.Index] = CollapseAll.Img;
+                
+                foreach (FDImage image in extensionIcons.Values)
+                {
+                    Images[image.Index] = image.Img;
+                }
+            }
+        }
```

Comment:
Here, it is much easier to re-assign all images instead of clearing and adding them in the correct order.